dspace-api/src/main/java/org/dspace/content/Process.java

```diff
@@ -0,0 +1,204 @@
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.dspace.core.ReloadableEntity;
+import org.dspace.eperson.EPerson;
+
+/**
+ * This class is the DB Entity representation of the Process object to be stored in the Database
+ */
+@Entity
+@Table(name = "process")
+public class Process implements ReloadableEntity<Integer> {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "process_id_seq")
+    @SequenceGenerator(name = "process_id_seq", sequenceName = "process_id_seq", allocationSize = 1)
+    @Column(name = "process_id", unique = true, nullable = false)
+    private Integer processId;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private EPerson ePerson;
+
+    @Column(name = "start_time")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date startTime;
+
+    @Column(name = "finished_time")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date finishedTime;
+
+    @Column(name = "script", nullable = false)
+    private String name;
+
+    @Column(name = "status")
+    @Enumerated(EnumType.STRING)
+    private ProcessStatus processStatus;
+
+    @Column(name = "parameters")
+    private String parameters;
+
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(
+        name = "process2bitstream",
+        joinColumns = {@JoinColumn(name = "process_id")},
+        inverseJoinColumns = {@JoinColumn(name = "bitstream_id")}
+    )
+    private List<Bitstream> bitstreams;
+
+    @Column(name = "creation_time", nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date creationTime;
+
+    protected Process() {
+    }
+
+    public Integer getID() {
+        return processId;
+    }
+
+    public void setProcessId(Integer processId) {
+        this.processId = processId;
+    }
+
+    public EPerson getEPerson() {
+        return ePerson;
+    }
+
+    public void setePerson(EPerson ePerson) {
+        this.ePerson = ePerson;
+    }
+
+    public Date getStartTime() {
+        return startTime;
+    }
+
+    public void setStartTime(Date startTime) {
+        this.startTime = startTime;
+    }
+
+    public Date getFinishedTime() {
+        return finishedTime;
+    }
+
+    public void setFinishedTime(Date finishedTime) {
+        this.finishedTime = finishedTime;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public ProcessStatus getProcessStatus() {
+        return processStatus;
+    }
+
+    public void setProcessStatus(ProcessStatus processStatus) {
+        this.processStatus = processStatus;
+    }
+
+    /**
+     * To get the parameters, use ProcessService.getParameters() to get a parsed list of DSpaceCommandLineParameters
+     */
+    protected String getParameters() {
```

Comment:
Please improve Javadocs. What is the `String` returned here?  Is it the string representation of the parameters (unparsed)?